playbooks/installation/installation.adoc

```diff
@@ -50,28 +50,32 @@ The Smart Start architecture requires the following Servers(VMs) be provisioned:
 
```

Comment:
line 20 above, second sentence, first word - replace "Two" with "Three" to reflect the number of infra nodes in the updated diagram.

line 30 above -  update '2 infrastructure...' to '3 infrastructure...'

line 45 above - replace 'has' with 'as' in the sentence "...Linux host has a load balancer..."